scripts/fdbshow

```diff
@@ -44,7 +44,7 @@ class FdbShow(object):
         self.if_name_map, \
         self.if_oid_map = port_util.get_interface_oid_map(self.db)
         self.if_br_oid_map = port_util.get_bridge_port_map(self.db)
-        self.fetch_fdb_data()
+        self.fetch_state_fdb_data()
         return
 
     def fetch_fdb_data(self):
```

Comment:
>fetch_fdb_data [](start = 8, length = 14)

Remove this function?

Reply:
I think we can remove it eventually but I don't think we should remove it right away.

Reply:
Since this function is still useful, can we make a CLI option to use this function if really needed.

---
In reply to: [449332606](https://github.com/Azure/sonic-utilities/pull/968#discussion_r449332606) [](ancestors = 449332606)